worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/SpongeSchematicReader.java

```diff
@@ -154,6 +155,12 @@ private CompoundTag getBaseTag() throws IOException {
         // Check
         Map<String, Tag> schematic = schematicTag.getValue();
 
+        // Handle newer versions in a cleaner way
```

Comment:
I would like to be more explicit about the leniency here.
```suggestion
        // Be lenient about the specific nesting level of the Schematic tag
        // Also allows checking the version from newer versions of the specification
```